website/static/js/tests/registrationUtils.test.js

```diff
@@ -0,0 +1,448 @@
+/*global describe, it, expect, example, before, after, beforeEach, afterEach, mocha, sinon*/
+'use strict';
+var assert = require('chai').assert;
+var $ = require('jquery');
+var utils = require('tests/utils');
+var faker = require('faker');
+
+var bootbox = require('bootbox');
+
+var $osf = require('js/osfHelpers');
+
+window.contextVars.currentUser = {
+    fullname: faker.name.findName(),
+    id: 1
+};
+var registrationUtils = require('js/registrationUtils');
+
+var utilities = registrationUtils.utilities;
+var Comment = registrationUtils.Comment; // jshint ignore:line
+var Question = registrationUtils.Question;
+var MetaSchema = registrationUtils.MetaSchema;
+var Draft = registrationUtils.Draft;
+var RegistrationEditor = registrationUtils.RegistrationEditor;
+var RegistrationManager = registrationUtils.RegistrationManager;
+
+var mkMetaSchema = function() {
```

Comment:
Please spell out variable names in favor of uncommon abbreviations. https://github.com/felixge/node-style-guide#use-lowercamelcase-for-variables-properties-and-function-names